downloads.html

```diff
@@ -16,6 +27,48 @@
   Have a look at our <a href="//wiki.minetest.net/Getting_Started">Getting Started</a>, <a href="//wiki.minetest.net/FAQ">FAQ</a> and <a href="//wiki.minetest.net/Tutorials">Tutorials</a> pages on our wiki.
 </p>
 
+<div class="intro" id="download-btn">
+    <a class="btn btn-success btn-lg" role="button">
+        Download Minetest {{ page.latest_version }}
+
+        <small>awaiting content</small>
+    </a>
+
+    <p>
+        <small>scroll down for more versions</small>
+    </p>
+</div>
+
+<script>
+    function setupDownloadBtn() {
+        var platform = navigator.platform.toLowerCase();
+        var platform_name = "<?>";
+        var title = "<?>";
+        var url;
+        if (platform.indexOf("win") >= 0) {
+            platform_name = "Windows";
+            title = "{{ page.latest_downloads.windows.title }}";
+            url = "{{ page.latest_downloads.windows.url }}";
+        } else if (platform.indexOf("mac") >= 0) {
+            platform_name = "OS X";
+            title = "{{ page.latest_downloads.mac.title }}";
+            url = "{{ page.latest_downloads.mac.url }}";
+        } else if (platform.indexOf("linux") >= 0) {
+            platform_name = "Ubuntu";
```

Comment:
**TRIGGERED**

I mean this is pretty stupid, most browsers on Ubuntu even have that in the user agent so you could look for that. Don't even bother trying to add a download button for the other linux flavors